crates/client/src/lib.rs

```diff
@@ -535,6 +555,12 @@ pub enum ClientError {
     #[error("no default registry server URL is configured")]
     NoDefaultUrl,
 
+    /// The operator failed validation.
+    #[error("operator failed validation")]
```

Comment:
Oh whoops, approved too soon; can you update the error message to `operator validation failed: {inner}`?